src/Documentation/ui/DocumentationRoot.tsx

```diff
@@ -15,19 +21,48 @@ export function DocumentationRoot({ docPage }: { docPage?: string }): React.Reac
   const history = useHistory();
   const [deepLink, setDeepLink] = useState(docPage);
   const navigator = {
-    navigate(relativePath: string, external: boolean) {
-      const path = relativePath.startsWith("nsDoc/")
-        ? asFilePath(relativePath)
-        : resolveFilePath("./" + relativePath, history.page);
+    navigate(href: string, external: boolean) {
```

Comment:
Optional: Rename the 2nd param to "openExternally" to make it clear that you're specifying the action, not a property of the href.